public/components/visualizations/charts/lines/line.tsx

```diff
@@ -81,6 +83,32 @@ export const Line = ({ visualizations, layout, config }: any) => {
       colorTheme.find((colorSelected) => colorSelected.name.name === field)?.color) ||
     PLOTLY_COLOR[index % PLOTLY_COLOR.length];
 
+  const checkIfMetrics = (metricsSchema: any) => {
```

Comment:
these functions seem to be able to be move out of line chart to be the general logics for checking metric schema.